experimental/dds/tree2/src/test/shared-tree-core/editManager.spec.ts

```diff
@@ -552,8 +552,7 @@ describe("EditManager", () => {
 						trunkEditCount * rebasedEditCount +
 							rebasedEditCount * (rebasedEditCount - 1),
 					);
-					// TODO: Task4664 Prevent quadratic number of inversions by caching inverses
-					// assert.equal(nbInverted, nbRebased - 1);
+
 					assert.equal(
```

Comment:
Sadly, we can't use `rebaser.invertedCount` to count meaningful inverts (since we're still calling invert, it's just not doing anything). @yann-achard-MS do you have any ideas for alternatives? I'm thinking the best thing to do is remove the assert here and replace it with a targeted test in modularChangeFamily.spec.ts that counts the number of unique objects returned by `invert`.

Reply:
I made the first approach and I don't think is necessary to remove this assert.